examples/graph/plot_DAG_layouts.py

```diff
@@ -0,0 +1,53 @@
+"""
+======================
+DAG in Various Layouts
+======================
+Creates a Directed Acyclic Graph and shows the Graph in various layouts. 
+This can give an idea as to which layout can be chosen for these types of datasets.
+"""
+
+import networkx as nx
+import matplotlib.pyplot as plt
+
+
+G = nx.DiGraph(
+    [
+        ("f", "a"),
+        ("a", "b"),
+        ("a", "e"),
+        ("b", "c"),
+        ("b", "d"),
+        ("d", "e"),
+        ("f", "c"),
+        ("f", "g"),
+        ("h", "f"),
+    ]
+)
+
+
+fig, axes = plt.subplots(2, 4, figsize=(25, 12))
```

Comment:
It might be worth experimenting (locally) with scaling the figsize back a little bit. The relationship between figure size and other components of the visualization (e.g. fontsize) can often be surprisingly intertwined. For example, reducing the figsize (while preserving roughly the same aspect ratio) may help make some of the titles/labels more readable.

Another thing worth trying out is adding `fig.tight_layout()` before `plt.show()` to try to eliminate as much of the unused space in the figure as possible.

Reply:
Thanks! I have adjusted the figsize by reducing it by half as of now and setting a size on the titles. I hope its alright. 😅 Please let me know if I can improve it better :)